src/main/java/core/basesyntax/FigureTypes.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum FigureTypes {
+    CIRCLE, ISOSCELES_TRAPEZOID, RECTANGLE, RIGHT_TRIANGLE, SQUARE
+}
```

Comment:
You can move this enam to FigureSupplier class, because you use it only in this class

Reply:
CI Tests don't allow saving enum in the same file class. Left it in the seperate file.